### PR TITLE
Add learning objectives and bug fixes to mnist_linear

### PR DESCRIPTION
**Pull request by danideter**
Making mnist_liear more consistent with mnist_models

### CODE REVIEW

File: courses/machine_learning/images/mnist_linear.ipynb [hunk 1]
```diff
@@ -9,7 +9,12 @@
    "source": [
```
Comment (BenoitDherin) [inline courses/machine_learning/images/mnist_linear.ipynb]: <ul><li>grep tensorflow==2 -&gt; grep tensorflow==2.0.0</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BenoitDherin) [inline courses/machine_learning/images/mnist_linear.ipynb]: <ul><li>move all imports at the beginning with the other "import tensorflow as tf" statement</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (danideter) [reply courses/machine_learning/images/mnist_linear.ipynb]: Done.

Comment (danideter) [reply courses/machine_learning/images/mnist_linear.ipynb]: Chatted with Benoit offline. Keeping libraries more general for easier maintenance.

**Comment by danideter** [comment]
Thanks for your awesome reviews as always, Benoit!

**Pull request merged**